apps/api/src/persistence/database/types/ArangoCollectionId.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * TODO [https://www.pivotaltracker.com/story/show/182132515]
+ * Refactor. Reorganize our collection name \ type management.
```

Comment:
Good call here, this was a bit hard to follow and understand the distinction between the resource collection ids and the other document (as opposed to edge) collection ids